### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by afdta**
I don't have a ton of experience with bundlers (I am using rollup) and I would have benefited from this quick explanation.

### CODE REVIEW

**Comment by domoritz** [comment]
Thank you for the pull request. However, I think the readme isn't the right place for this note. Instead, it should be at https://vega.github.io/vega/usage/.

**Comment by afdta** [comment]
> Thank you for the pull request. However, I think the readme isn't the right place for this note. Instead, it should be at https://vega.github.io/vega/usage/.

I guess I'm confused, I thought this was the markdown for the usage page?

**Comment by domoritz** [comment]
Ignore me. I should go to bed. You are right.

File: docs/usage/index.md [hunk 1]
```diff
@@ -53,6 +53,12 @@ To use Vega on a web page you first need to load the Vega JavaScript files. The
 </head>
 ```
 
+**Using Vega with a bundler.** If using Vega with a package manager (e.g. [npm](https://www.npmjs.com/package/vega)) and a bundler like [rollup.js](https://rollupjs.org/guide/en/#with-npm-packages), you can import everything into a namespace as follows:
```
Comment (domoritz) [inline docs/usage/index.md:56]:
```suggestion
**Using Vega with a bundler.** If you use Vega with a bundler like [rollup.js](https://rollupjs.org/guide/en/#with-npm-packages), you can import Vega [as a module](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import).
```

**Review by domoritz — APPROVED** [review]

**Comment by jheer** [comment]
Thanks @afdta!

**Pull request merged**